docs/catalog/sinks/telegram.rst

```diff
@@ -0,0 +1,78 @@
+Telegram
+#################
+
+Robusta can send playbook results to Telegram conversations.
+
+.. note::
+
+    ``TableBlock`` is omitted from Telegram sinks because it's too big for most Telegram chat clients.
```

Comment:
I would say `tables` here not `TableBlock` as most users reading this aren't familiar with the code or the findings API.

Reply:
fixed